bundles/org.openhab.binding.caddx/src/main/java/org/openhab/binding/caddx/internal/handler/LogEventMessage.java

```diff
@@ -56,33 +56,34 @@ public String toString() {
             int eventType = Integer.parseInt(type);
             logger.trace("eventType received: {}", eventType);
             LogEventType logEventType = LogEventType.valueOfLogEventType(eventType);
-            if (logEventType == null) {
-                return "Unknown log event type received";
-            }
 
             // Date
             sb.append(String.format("%02d", Integer.parseInt(day))).append('-')
                     .append(String.format("%02d", Integer.parseInt(month))).append(' ')
                     .append(String.format("%02d", Integer.parseInt(hour))).append(':')
                     .append(String.format("%02d", Integer.parseInt(minute))).append(' ');
 
-            sb.append(logEventType.description);
-            if (logEventType.isPartitionValid) {
-                sb.append(" Partition ").append(Integer.parseInt(partition) + 1);
-            }
+            if (logEventType == null) {
+                sb.append("Unknown log event type");
+            } else {
+                sb.append(logEventType.description);
+                if (logEventType.isPartitionValid) {
+                    sb.append(" Partition ").append(Integer.parseInt(partition) + 1);
+                }
 
-            switch (logEventType.zud) {
-                case None:
-                    break;
-                case Zone:
-                    sb.append(" Zone ").append(Integer.parseInt(zud) + 1);
-                    break;
-                case User:
-                    sb.append(" User ").append(Integer.parseInt(zud) + 1);
-                    break;
-                case Device:
-                    sb.append(" Device ").append(zud);
-                    break;
+                switch (logEventType.zud) {
+                    case None:
```

Comment:
These should be all upper case. Would be awesome if you could file a follow-up PR!

Reply:
Yes. will do.